man/marked.1.txt

```diff
@@ -26,71 +19,76 @@ EXAMPLES
 
-
 OPTIONS
        -o, --output [output]
-	      Specify file output. If none is specified, write to stdout.
+              Specify file output. If none is specified, write to stdout.
 
        -i, --input [input]
-	      Specify file input, otherwise use last argument as  input  file.
-	      If no input file is specified, read from stdin.
+              Specify file input, otherwise use last argument as input file. If no input file is specified, read from stdin.
+
+       --test Makes sure the test(s) pass.
+
+              --glob [file] Specify which test to use.
+
+              --fix Fixes tests.
+
+              --bench Benchmarks the test(s).
+
+              --time Times The test(s).
+
+              --minified Runs test file(s) as minified.
+
+              --stop Stop process if a test fails.
 
        -t, --tokens
-	      Output a token stream instead of html.
+              Output a token stream instead of html.
 
        --pedantic
-	      Conform  to  obscure  parts  of markdown.pl as much as possible.
-	      Don't fix original markdown bugs.
+              Conform to obscure parts of markdown.pl as much as possible. Don't fix original markdown bugs.
 
        --gfm  Enable github flavored markdown.
 
        --breaks
-	      Enable GFM line breaks. Only works with the gfm option.
+              Enable GFM line breaks. Only works with the gfm option.
 
        --tables
-	      Enable GFM tables. Only works with the gfm option.
+              Enable GFM tables. Only works with the gfm option.
 
        --sanitize
-	      Sanitize output. Ignore any HTML input.
+              Sanitize output. Ignore any HTML input.
 
        --smart-lists
-	      Use smarter list behavior than the original markdown.
+              Use smarter list behavior than the original markdown.
 
        --lang-prefix [prefix]
-	      Set the prefix for code block classes.
+              Set the prefix for code block classes.
 
        --mangle
-	      Mangle email addresses.
+              Mangle email addresses.
 
        --no-sanitize, -no-etc...
-	      The inverse of any of the marked options above.
+              The inverse of any of the marked options above.
 
        --silent
-	      Silence error output.
+              Silence error output.
 
        -h, --help
-	      Display help information.
-
+              Display help information.
 
 CONFIGURATION
        For configuring and running programmatically.
 
        Example
 
-	   require('marked')('*foo*', { gfm: true });
-
+           require('marked')('*foo*', { gfm: true });
 
 BUGS
        Please report any bugs to https://github.com/markedjs/marked.
 
-
 LICENSE
        Copyright (c) 2011-2014, Christopher Jeffrey (MIT License).
 
-
 SEE ALSO
        markdown(1), node.js(1)
 
-
-
-v0.3.1				  2014-01-31			     marked(1)
+v0.3.1                                                             2014-01-31                                                          marked(1)
```

Comment:
@styfle Should this version and date be updated any time we release a new version?

Reply:
and maybe the line shouldn't be so long?

Reply:
Maybe we should remove the version from the man page and instead provide a `--version` flag.